docs/UserGuide.md

```diff
@@ -198,6 +199,16 @@ Examples:
 * `find alex` followed by `delete 1` deletes the 1st contact in the results of the `find` command.
   ![result for 'delete 1 after finding alex'](images/deleteAlexYeohResult.png)
 
+#### Delete by name
+Format `delete NAME` or `delete [n/NAME]`
+
+* Delete the contact whose name matches the specified `NAME`.
```

Comment:
Should we make it more clear that it has to be an exact match? since find uses substring matching

Reply:
Oh yes, thanks.